docs/pages/configuration/pipelines/functions/get_config_value.mdx

```diff
@@ -2,3 +2,17 @@
 title: Function get_config_value
 sidebar_label: get_config_value
 ---
+
+Description needed here
```

Comment:
If this is an intentional note, can you add TODO in front of it so I know?

Reply:
fixed